packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/SummarySheet.tsx

```diff
@@ -107,28 +112,49 @@ const SummarySheet = (props: Props) => {
                       <tbody>
                         <tr>
                           <td align='center' width='100%'>
-                            <div className='flex justify-center gap-4'>
+                            <div className='mb-2 flex justify-center gap-4'>
```

Comment:
I added this margin to give the buttons more room to breathe: 

![Screenshot 2023-09-18 at 19 36 15](https://github.com/ParabolInc/parabol/assets/39854876/b0a6601a-0478-435f-8c73-c1428d857ed7)

Reply:
+1 what do you think about hiding these when they're printed? just add the class `.hide-print` to the parent
you could also make it take up the full page width if you wanted to get fancy (e.g. for an A4 letter size, add the class `print:w-[210mm]`. tailwinds is awesome, basically add the `print:` prefix to anything & it'll only show up in print mode!

Reply:
Oh that's cool! Didn't know you could do that with tailwind. Updated!